### ISSUE TITLE
the "test" dataset must have label? 

### ISSUE THREAD

**Issue by JINGZIjingzi** [mravanelli/pytorch-kaldi#201]
Since I am using the Babel dataset for keyword search, the eval set (here, the test set) doesn't have the corresponding transcription text. So I can't get the ali-files (ali1.gz,ali2.gz,……)which should be in "lab_folder". what should I do to solve this……
If I use the dev dataset for test, everthing looks good. But I need the eval set to test or decode……

**Comment by TParcollet** [comment]
You can decode without the labels following the "How can I transcript my own audio files" part of the readme. Nonetheless, you won't be able to use the test set without labels as a validation set since you need a loss metric that is computed w.r.t the labels.

**Comment by JINGZIjingzi** [comment]
Thank you for your quick reply! I will try
